docs/modules/ROOT/pages/guides/v4-migration/authorization.adoc

```diff
@@ -0,0 +1,272 @@
+= Authentication and Authorization
+
+The largest breaking change in version 4.0.0 is the removal of the `@auth` directive, which will require a migration to the new `@authentication` and `@authorization` directives.
+
+== Instantiation
+
+Whilst the `@auth` directive required the installation of an additional plugin package, the functionality for the new directives is built directly into the library. You should uninstall the previous plugin:
+
+[source, bash, indent=0]
+----
+npm uninstall @neo4j/graphql-plugin-auth
+----
+
+Then, given an example of instantiation using a basic secret with the plugin:
+
+[source, typescript, indent=0]
+----
+new Neo4jGraphQL({
+    typeDefs,
+    plugins: {
+        auth: new Neo4jGraphQLAuthJWTPlugin({
+            secret: "secret",
+        }),
+    }
+})
+----
+
+You can delete the import of `Neo4jGraphQLAuthJWTPlugin` and change the instantiation to:
+
+[source, typescript, indent=0]
+----
+new Neo4jGraphQL({
+    typeDefs,
+    features: {
+        authorization: {
+            key: "secret",
+        }
+    }
+})
+----
+
+== Server
+
+Previously, you could pass in the entire request object and the library would fish the `Authorization` header out:
+
+[source, typescript, indent=0]
+----
+const server = new ApolloServer({
+    schema, // schema from Neo4jGraphQL.getSchema()
+});
+
+const { url } = await startStandaloneServer(server, {
+    listen: { port: 4000 },
+    context: async ({ req }) => ({ req }),
+});
+----
+
+With the new implementation, the library expects the `Authorization` header to be extracted in the format of a bearer token in the `token` field of the context:
```

Comment:
Should "bearer". be capitalized?

Reply:
No, it's only capitalized when used inside a HTTP header as far as I'm aware!